### DISCUSSION TITLE
守信用乃商业之根本：从开源世界历史上的一场官司说起 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#109] [Announcements]
# 守信用乃商业之根本：从开源世界历史上的一场官司说起 - 「开源之道」

https://opensourceway.community/posts/open-source-economic/business-care-about-criedt-in-os-community/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

> 没有人知道广告有没有用，但是就是需要去做的事情...

真相了, 其实

> ..部分对商业有曲解的人士认为，凡是没有法律介入的都可以肆无忌惮的侵害，比如GPL许可下的软件项目...

这种主动+故意的曲解, 就是"聪明人"值得去作的事儿之一, 和

> 打瞎子、骂哑巴，踢寡妇门、挖绝户坟

是同类行为, 虽然为正常人不齿, 但是, 总是有 "聪明人" 去作, 毕竟正常法律秩序中并没有专门的部门管这种事儿.

是的, FLOSS 项目缺少一个明确的社会法人形象来以眼还眼以牙还牙, 自然就变成了 "瞎子/哑巴/寡妇/绝户坟" ...

这不是 OSPO 可以解决的事儿, 得要每个理解 FLOSS 的人, 主动站出来说点儿什么, 作点儿什么...

**Reply by lijiangsheng1** [reply]
我们无法要求他人做事～ 只能自己做点力所能及的事～这也是倡导亲力亲为的重要依据。
